onecodex/models/collection.py

```diff
@@ -350,7 +353,10 @@ def _collate_results(self, metric=None, include_host=None):
                 if not include_host and d_tax_id in host_tax_ids:
                     continue
 
-                data[c_idx, tax_id_to_idx[d_tax_id]] = d[metric] or 0
+                if metric in [Metric.AbundanceWChildren, Metric.Abundance]:
+                    data[c_idx, tax_id_to_idx[d_tax_id]] = d[metric]
```

Comment:
Should we use `d[metric] or np.nan` here?

Reply:
I think it shouldn't actually make a difference, but for consistency it is probably better

Reply:
Hmm but then `0.0` or `0` abundances will get coerced into `np.nan`. Is that something we want?

Reply:
Ah, good point - no, we want to keep 0 and 0.0. I will revert to d[metric] in this case